coremltools/models/deprecation.py

```diff
@@ -0,0 +1,23 @@
+import warnings
+import functools
+
+def deprecated(func=None, str=""):
+    '''
+    Decorator to mark a function or class as deprecated
+    '''
+
+    def decorator_deprecation_warning(func):
+        @functools.wraps(func)
+        def wrapped(*args, **kwargs):
+            msg = "Function %s will be deprecated from version 4" % func.__name__
```

Comment:
I would probably change this error message to:
`"The \"%s\" function is deprecated and will be removed in the next release"`

Reply:
Updated